internal/service/thirdpartyintegration/resource_third_party_integration_test.go

```diff
@@ -92,21 +109,21 @@ func TestAccConfigRSThirdPartyIntegration_updateBasic(t *testing.T) {
 	resource.Test(t, resource.TestCase{
 		PreCheck:                 func() { acc.PreCheck(t) },
 		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
-		CheckDestroy:             testAccCheckMongoDBAtlasThirdPartyIntegrationDestroy,
+		CheckDestroy:             checkDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccMongoDBAtlasThirdPartyIntegrationResourceConfig(&seedInitialConfig),
+				Config: configBasic(&seedInitialConfig),
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckThirdPartyIntegrationExists(resourceName, &targetIntegration),
+					checkExists(resourceName),
 					resource.TestCheckResourceAttr(resourceName, "type", cfg.Type),
 					resource.TestCheckResourceAttr(resourceName, "api_key", cfg.APIKey),
 					resource.TestCheckResourceAttr(resourceName, "region", cfg.Region),
 				),
 			},
 			{
-				Config: testAccMongoDBAtlasThirdPartyIntegrationResourceConfig(&seedUpdatedConfig),
+				Config: configBasic(&seedUpdatedConfig),
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckThirdPartyIntegrationExists(resourceName, &targetIntegration),
+					checkExists(resourceName),
```

Comment:
[q] Shouldn't the migration test have a step where the latest provider release version is used? 🤔

Reply:
that's done now with helper function, e.g.:

mig.CreateAndRunTest(t, basicTestCase(t))